frontend/packages/topology/src/components/DefaultCreateConnector.tsx

```diff
@@ -1,40 +1,66 @@
 import * as React from 'react';
-import Point from '../geom/Point';
+import * as classNames from 'classnames';
+import { AddCircleOIcon } from '@patternfly/react-icons';
+import { Tooltip } from '@patternfly/react-core';
 import ConnectorArrow from './ConnectorArrow';
+import { CreateConnectorRenderer } from './CreateConnectorWidget';
 
 import './DefaultCreateConnector.scss';
 
-type DefaultCreateConnectorProps = {
-  startPoint: Point;
-  endPoint: Point;
-  hints?: string[];
-};
+const cursorSize = 20;
 
-const DefaultCreateConnector: React.FC<DefaultCreateConnectorProps> = ({
+const DefaultCreateConnector: CreateConnectorRenderer = ({
```

Comment:
You'll need a new prop type that includes support for `tipContents` and `className`.